doc/classes/Node2D.xml

```diff
@@ -156,6 +156,10 @@
 		<member name="z_index" type="int" setter="set_z_index" getter="get_z_index" default="0">
 			Z index. Controls the order in which the nodes render. A node with a higher Z index will display in front of others.
 		</member>
+		<member name="y_sort_enabled" type="bool" setter="set_y_sort_enabled" getter="is_y_sort_enabled" default="true">
+			If [code]true[/code], child nodes with the lowest Y position are drawn before those with a higher Y position. If [code]false[/code], Y-sorting is disabled. Y-sorting only affects children that inherit from [CanvasItem].
+			You can nest nodes with Y-sorting. Children Y-sorted nodes are sorted in the same space as the parent Y-sort. This feature allows you to organize a scene better or divide it into multiple ones without changing your scene tree.
+		</member>
```

Comment:
Those are normally ordered alphabetically, so it could come before `z_as_relative`. Did you add it here manually or did `doctool` place it there? (in which case that would be a bug)

Reply:
I have fixed this by actually using doctool, didn't even know that existed!